### ISSUE TITLE
[BUG]: vitest-accessibility-checker exports `eRuleLevel` as a type so it cannot be used as an enum

### ISSUE THREAD

**Issue by samueldmeyer** [IBMa/equal-access#2605]
### Project

other

### Description

`eRuleLevel` is internally an `enum` in TypeScript, but `index.d.ts` only re-exports it as a `type`, so it cannot be used as an `enum` by TypeScript consumers. As a result, consumers must cast it to the correct type. Instead, it should just be exported without `type`.
